README.Rmd

```diff
@@ -68,7 +68,9 @@ To get started, read the tidy data vignette (`vignette("tidy-data")`) and check
 
 ## Related work
 
-tidyr replaces reshape2 (2010-2014) and reshape (2005-2010). Somewhat counterintuitively each iteration of the package has done less. tidyr is  designed specifically for tidying data, not general reshaping (reshape2), or the general aggregation (reshape).
+tidyr replaces reshape2 (2010-2014) and reshape (2005-2010). Somewhat counterintuitively each iteration of the package has done less. tidyr is designed specifically for tidying data, not general reshaping (reshape2), or the general aggregation (reshape). 
+
+[data.table](http://r-datatable.com/) provides high performance implementations of `melt()` and `dcast()`
```

Comment:
Do we have links to `melt()` and `dcast()` docs for data.table? If so, we can add.

Reply:
No, there's no pkgdown website yet.